pages/platform/databases/m3dbaggregator_01_capabilities/guide.de-de.md

```diff
@@ -0,0 +1,123 @@
+---
+title: M3db Aggregator - Capabilities and Limitations
```

Comment:
I think it's called M3 Aggregator, not M3db Aggregator: see e.g. https://m3db.io/docs/architecture/m3aggregator/, https://aiven.io/m3-aggregator

Reply:
Fixed